templates/admin-sdk.html.twig

```diff
@@ -6,12 +6,17 @@
 
     <title>Braintree</title>
 
-    {{ encore_entry_link_tags('admin') }}
-    {{ encore_entry_script_tags('admin') }}
+    {% if hmr %}
+    <script type="module" rel="script" src="/_vite_/@vite/client"></script>
+    <script type="module" rel="script" src="/_vite_/assets/admin.ts"></script>
+    {% else %}
+    <link rel="stylesheet" href="/build/admin.css">
+
+    <script type="module" src="/build/admin.js"></script>
+    <script type="module" src="/build/polyfills.js"></script>
```

Comment:
could we make these paths subdirectory-able?

Reply:
We could, at least in my development environment this works - just for the record, the encore build before also only outputs absolute paths